docs/field-descriptions.yaml

```diff
@@ -91,6 +98,14 @@ health_status: "Patient's baseline physical condition prior to immediate event l
   \ \u201Crelatively healthy.\u201D"
 height_unit: The vertical measurement or distance from the base to the top of a subject
   or participant.
+internal_standard: Specifies if internal standard was utilized and designates what
+  standard was used. Leave blank if not applicable.
+ion_mobility: 'Specifies whether or not ion mobility spectrometry was performed and
```

Comment:
So it this a boolean or a value?  Both?   Not so familiar with these docs but if there is some sort of constraint, it's not super obvious.

Reply:
I can see from below that this is not a boolean so take this comment or leave it.